src/app/auth/auth-oidc-config/oidc-auth.interceptor.ts

```diff
@@ -5,13 +5,16 @@ import { Observable } from 'rxjs';
 import { environment } from 'src/environments/environment';
 
 @Injectable()
-export class WharfAuthInterceptor implements HttpInterceptor {
+export class OidcAuthInterceptor implements HttpInterceptor {
 
   constructor(
     private oidcSecurityService: OidcSecurityService,
   ) { }
 
   intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+    if (!environment.oidcConfig?.enabled) {
+      return next.handle(req);
+    }
     const apiUrl = environment.backendUrls.api;
     if (!req.url.includes(apiUrl)) {
       return next.handle(req);
```

Comment:
We probably never want to intercept URLs that don't _start with_ the `apiUrl`.
I believe we had a short discussion about this before, so pointing it out again:

Probably best to change to using `if (!req.url.startsWith(apiUrl)) {` here, just in case.

Reply:
Yea, totally agree.

I've used your suggestion, and also added checks if it's one of the importer's URLs as well.